data/data.go

```diff
@@ -0,0 +1,422 @@
+// Package data implements sql operations over the NoName e.V. website-db.
+package data
+
+import (
+	"database/sql"
+	"errors"
+	"flag"
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var (
+	driver  = flag.String("driver", "postgres", "Der benutzte sql-Treiber")
+	connect = flag.String("connect", "dbname=nnev user=anon host=/var/run/postgresql sslmode=disable", "Die Verbindusgsspezifikation")
+)
+
+// OpenDB opens a connection to the database with parameters derived from flags.
+func OpenDB() (*sql.DB, error) {
+	return sql.Open(*driver, *connect)
+}
+
+// Querier is an interface used to query the database. Both *sql.DB and *sql.Tx
```

Comment:
I'd go for at least a little longer lines to avoid multiline in the comment here.